db/item_db.conf

```diff
@@ -150872,6 +150872,27 @@ item_db: (
 
 //== New Etc Items =========================================
 {
+	Id: 22508
+	AegisName: "Para_Team_Mark_"
+	Name: "Eden Group Mark"
+	Type: 11
+	Buy: 0
+	Delay: 1200000
+	Trade: {
+		nodrop: true
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nostorage: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		unitskilluseid getcharid(3),"AL_TELEPORT",3;
```

Comment:
You are right. I found a note from Ziu in 1/27/2016 - Maintenance topic on iRO Wiki Forums:

Only kRO servers.
paradise group mark - eden Group Mark (iro)

-> Before: Delay 30 min, double click and warp last memo point map.
-> Now : Delay 20 Min, double click and warp to 2 floor in paradise group
(NoTe - if you walk out the warp (paradise group - city), you go to the city where last time you accessed to paradise group.).

Source: http://forums.irowiki.org/index.php?topic=107188.msg1412792#msg1412792